dbt_common/events/logger.py

```diff
@@ -120,7 +122,11 @@ def create_line(self, msg: EventMsg) -> str:
     def write_line(self, msg: EventMsg):
         line = self.create_line(msg)
         if self._python_logger is not None:
-            send_to_logger(self._python_logger, msg.info.level, line)
+            if isinstance(msg.data, PrintEvent):
```

Comment:
@peterallenwebb This is a draft to help me understand some way of achieving the goal. Let's discuss Monday and see what you think.

Reply:
related PR https://github.com/dbt-labs/dbt-core/pull/10131

Reply:
This looks like the best way to achieve what we want, the only drawback is we can not have core inherent this event and define new ones since the proto namespace will change.